src/rgw/rgw_common.cc

```diff
@@ -724,6 +727,9 @@ static inline bool check_deferred_bucket_acl(struct req_state *s, uint8_t deferr
 
 bool verify_object_permission(struct req_state *s, RGWAccessControlPolicy *bucket_acl, RGWAccessControlPolicy *object_acl, int perm)
 {
+  if ((s->auth_method).get_acl_main_override())
+    return true;
```

Comment:
Add log line that ACL decision will be overridden. Use DSS INFO tag so that we know its ours.